src/main/java/fi/hsl/transitdata/stopestimates/PubtransStopEstimatesFactory.java

```diff
@@ -89,8 +89,7 @@ private static boolean validateCommon(PubtransTableProtos.Common common) {
             return false;
         }
         if (common.getType() == 0) {
-            log.info("Event is for a via point, message discarded");
-            return false;
+            log.info("Event is for a via point");
```

Comment:
```suggestion
            log.debug("Event is for a via point");
            return false;
```

Reply:
Log statement is now:
`log.debug("Event is for a via point, message discarded");`